schemast/annotation.go

```diff
@@ -186,6 +192,31 @@ func entSQL(annot schema.Annotation) (ast.Expr, bool, error) {
 	return c, true, nil
 }
 
+func entGQL(annot schema.Annotation) (ast.Expr, bool, error) {
+	m := &entgql.Annotation{}
+	if err := mapstructure.Decode(annot, m); err != nil {
+		return nil, false, err
+	}
+	var c *ast.CallExpr
+	if m.MutationInputs != nil && len(m.MutationInputs) > 0 {
+		args := []ast.Expr{}
+		for _, input := range m.MutationInputs {
+			if input.IsCreate {
+				args = append(args, fnCall(selectorLit("entgql", "MutationCreate")))
+			} else {
+				args = append(args, fnCall(selectorLit("entgql", "MutationUpdate")))
+			}
+		}
+		c = fnCall(
+			selectorLit("entgql", "Mutations"), args...,
+		)
+	} else {
```

Comment:
`entgql.Annotation` has many more fields. With the current implementation, users may provide input annotations with unsupported fields and we will silently ignore these options. I think when building infra code like this, correctness is of utmost importance. 

Can you either:
1. Add support for the missing fields
2. Return an error in case an unsupported field is used?

Reply:
Agree and will add all the missing fields in the upcoming days.